site/src/api/typesGenerated.ts

```diff
@@ -292,6 +292,19 @@ export const BypassRatelimitHeader = "X-Coder-Bypass-Ratelimit";
 // From codersdk/client.go
 export const CLITelemetryHeader = "Coder-CLI-Telemetry";
 
+// From codersdk/workspacebuilds.go
+export interface CancelWorkspaceBuildParams {
+	readonly expect_status?: CancelWorkspaceBuildStatus;
```

Comment:
I can be wrong, but I think when we add a comment to the struct, or its props in codersdk package, it gets included in the TS types as well. I think lefting a comment in the `CancelWorkspaceBuildParams.ExpectStatus` could be helpful to understand when and why use it.

Reply:
I left a comment in codersdk, but unfortunately it doesn't get passed to the TS types 😞